test/config/application.rb

```diff
@@ -0,0 +1,115 @@
+# Initialize a dummy application that is required to test
+# the gem that supplies some behavior to another rails application
+
+#
+# Rails
+#
+
+require "rails"
```

Comment:
@nkumeiko 
I am not sure if wee need all the libraries from the list. Please, reduce to the minimum.